src/credit.ts

```diff
@@ -237,6 +238,25 @@ export default class Credit extends AinftBase {
     return this.sendRequest(HttpMethod.POST, trailingUrl, body);
   }
 
+  /**
+   * You can get lockup list of user 
```

Comment:
minor) period at the end?

Reply:
minor) how about just "Gets a user's lockup list."?

Reply:
Description mentioned in the second comment has been updated. Thanks!

What does the first comment mean??

Reply:
Ah, period(.) mark. (It's an obsolete comment now though.)

Reply:
Oh I got it. Thanks!